api/authorization.rst

```diff
@@ -43,25 +45,46 @@ __ https://github.com/commerceguys/guzzle-oauth2-plugin
 Obtain access token
 ~~~~~~~~~~~~~~~~~~~
 
-Send the request with the following parameters
+Send the request with the following parameters:
+
+Definition
+..........
 
-client_id
-    Public client id
-client_secret
-    Client secret
-grant_type
-    We will use 'password' to authorize as user
-username
-    User name
-password
-    User password
+.. code-block:: text
+
+    GET /oauth/v2/token
+
++---------------+----------------+--------------------------------------------------------------------------------------------------+
+| Parameter     | Parameter type | Description                                                                                      |
++===============+================+==================================================================================================+
+| client_id     | query          | Client id generated in a previous step                                                           |
++---------------+----------------+--------------------------------------------------------------------------------------------------+
+| client_secret | query          | Client secret generated in a previous step                                                       |
++---------------+----------------+--------------------------------------------------------------------------------------------------+
+| grant_type    | query          | We will use 'password' to authorize as user. Other available options are token and refresh-token |
++---------------+----------------+--------------------------------------------------------------------------------------------------+
+| username      | query          | User name                                                                                        |
++---------------+----------------+--------------------------------------------------------------------------------------------------+
+| password      | query          | User password                                                                                    |
++---------------+----------------+--------------------------------------------------------------------------------------------------+
 
 Example
 .......
 
-.. code-block:: text
+.. code-block:: bash
 
-    GET /oauth/v2/token?client_id=4fugw85qtim8c48s88g00s0gc040s4s4k8wk4kswowkksg0skw&client_secret=s8tg7026wo0k0koco8o4wc0kwocowskgwkk0wowsgc8o4408c&grant_type=password&username=sylius@example.com&password=sylius
+    curl http://demo.sylius.org/oauth/v2/token
+        -d "client_id"=demo_client
+        -d "client_secret"=secret_demo_client
+        -d "grant_type"=password
+        -d "username"=api@example.com
+        -d "password"=api
+
+.. tip::
+
+    In a developer environment there is a default API user and client data. To use this credentials you have to load `API`__ data fixtures. Otherwise you have to replace username, password with your data and client id with client secret with data generated in a previous step.
```

Comment:
``Otherwise you have to replace username, password with your data and client id with client secret with data generated in a previous step.`` This really incomprehensible, you should think about changing it or splitting in two sentences or whatever :smile: